### PR TITLE
load choices for version selector from json

### PR DESCRIPTION
**Pull request by voorloopnul**

### CODE REVIEW

File: templates/version_selector.html [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
+            }
+
+            opt.value = entry[0];
+            opt.innerHTML = entry[1];
+            select.appendChild(opt);
+        })
+    }
 </script>
+
+<script src="{{config.extra.version_choices}}"></script>
```
[the remark this answers is not in the document]
Comment (voorloopnul) [reply templates/version_selector.html]: I updated it to load pure data.

File: templates/version_selector.html [hunk 2]
```diff
@@ -1,15 +1,42 @@
...
-    <option {% if "/docs/1.2" in config.base_url %} selected {% endif %} value="/docs/1.2/">1.2</option>
-    <option {% if "/docs/1.1" in config.base_url %} selected {% endif %} value="/docs/1.1/">1.1</option>
-    <option {% if "/docs/1.0" in config.base_url %} selected {% endif %} value="/docs/1.0/">1.0</option>
+        class="w-32 mt-1 bg-blue-mariner border border-transparent text-white text-sm rounded-xl focus:ring-transparent focus:border-transparent block">
 </select>
 
 <script>
+    fetch("{{ config.extra.version_choices | safe }}").then(res => res.json())
+    .then(data => {
+        fetch_version_list(data)
```
Comment (ES-Alexander) [inline templates/version_selector.html:9]: Today I learnt about JavaScript function hoisting… trippy…

File: theme.toml [hunk 3]
```diff
@@ -27,6 +27,10 @@ include_content = true
 # The place where the logo in the navbar should link to
 logo_redirect_to = "/docs/"
 
+# If you enable this option, a select widget will be available to switch between different versions.
+# You need to provide an url for a jsonp file, an example can be found in the bluetheme docs.
```
Comment (ES-Alexander) [inline theme.toml:31]:
```suggestion
# You need to provide a url for a json file, an example can be found in the bluetheme docs.
```

**Review by ES-Alexander — APPROVED** [review]
Minor typos in a comment, but otherwise looks good - nice work! :-D

**Pull request merged**